apps/nextra/pages/en/build/sdks/wallet-adapter/dapp.mdx

```diff
@@ -1,206 +1,474 @@
-users can interact with the Aptos blockchain. Implementing wallet integration
-can be difficult in supporting all edge cases, new features, unsupported
-functionality. And it can be even harder to support multiple wallets.
+Aptos provides a React `Provider` and `Context` for connecting Aptos wallets to your dapp. This `Provider` allows you to specify which Wallets you want to allow connections to. Then you can use the `Provider` to look up account information and sign transactions / messages. 
 
-In addition, different wallets have different APIs, and not all wallets share
-the same naming convention. For example, maybe all wallets have a `connect`
-method, but not all wallets call that method `connect`; that can be tricky to
-support.
+This provides a standard interface for using all Aptos wallets, and allows new wallets to easily be supported just by updating your React Wallet Adapter dependency version. 
 
-Luckily, Aptos built a wallet adapter, created and maintained by the Aptos team,
-to help you ramp up development and standardize where possible.
+## Using the React `Provider` and `Context`
 
-The Aptos Wallet Adapter provides:
+<Steps>
 
-- Easy wallet implementation - no need to implement and support code for
-multiple wallets.
-- Support for different wallet APIs.
-- Support for features not implemented on the wallet level.
-- Detection for uninstalled wallets (so you can show users that a wallet is not
-installed).
-- Auto-connect functionality and remembers the current wallet state.
-- Listens to wallet events, such as account and network changes.
-- A well-developed and maintained reference implementation by the Aptos
-ecosystem team.
-
-## Install
-
-Currently, the adapter supports a _React provider_ for you to include in your
-app.
-
-Install wallet dependencies you want to include in your app. You can find a list
-of the wallets in the Aptos Wallet
-Adapter [README](https://github.com/aptos-labs/aptos-wallet-adapter#supported-wallet-packages).
-
-Install the React provider:
+### Install `@aptos-labs/wallet-adapter-react`.
 
 ```bash filename="Terminal"
 npm install @aptos-labs/wallet-adapter-react
 ```
 
-## Import dependencies
-
-In the `App.jsx` file:
+### (Optional) Install the plugins plugins for any “Legacy Standard Compatible” Wallets you want to support from [this list](https://github.com/aptos-labs/aptos-wallet-adapter/blob/main/README.md#supported-wallet-packages).
 
-Import the installed wallets:
+<Callout type="info">
+The more modern [AIP-62](https://github.com/aptos-foundation/AIPs/blob/main/aips/aip-62.md) wallets do NOT require installing a package - they work by default!
+</Callout>
 
-```jsx
-import { SomeWallet } from "some-wallet-plugin-wallet-adapter";
-```
+The legacy standard required installing plugins manually. For wallets that have not updated to using the AIP-62 standard, their plugins must be installed and passed in to the `Provider` manually.
```

Comment:
can we have it in a nice info box?

Reply:
Moved the legacy standard note into the above info box. The note about providing plugins manually is part of the steps though so I kept those in the "main" text area as they are instructions.